### PR TITLE
kate wanted this removed

### PR DESCRIPTION
**Pull request by LaurenSlu**
just did a quick removal of the menu scroll bar that was appearing for no reason at all

### CODE REVIEW

File: src/app/app.scss [hunk 1]
```diff
@@ -111,3 +111,7 @@ ion-item{
     display: flex;
     align-self: center;
 }
+
+.scroll-content {
+    overflow: hidden;
```
Comment (kjrcda) [inline src/app/app.scss]: try initial or auto, we want the scrollbar to show if there are more items than what fits on the page

**Comment by Yexur** [comment]
Blame Kate!!!!

**Comment by LaurenSlu** [comment]
@kjrcda switched it to auto. legit legit, seems to still work

**Comment by kjrcda** [comment]
Looks good, thanks! Must have been set to visible somewhere

**Pull request merged**